test/libsolidity/syntaxTests/types/empty_tuple_event.sol

```diff
@@ -2,9 +2,9 @@ pragma solidity ^0.4.3;
 contract C {
     event SomeEvent();
     function a() public {
-        (SomeEvent(), 7);
+        (emit SomeEvent(), 7);
```

Comment:
This with `emit SomeEvent` is not parsed correctly - "expected primary expression". Either we can just remove the test case or "fix" the parser.

Reply:
We remove the test case or require it to fail. I specifically wrote the parser to only accept `emit` at statement level.

Reply:
Or was that only changed later? Anyway, EmitStatements should be statements :)